packages/frontend/components/ui/checkbox.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Checkbox({
+  name,
+  checked,
+  type,
+  disabled,
+  className,
+  value,
+  label,
+  required,
+  onChange,
+  ...otherProps
+}) {
+  function changeValue(targetValue) {
+    if (onChange) {
+      onChange(targetValue);
+    }
+  }
+
+  let customStyleName = '';
+  if (checked && type === 'success') {
+    customStyleName = 'success-check-box';
+  } else if (checked && type === 'danger') {
+    customStyleName = 'danger-check-box';
+  }
+
+  return (
+    <div className="flex items-center">
+      <input
+        id={name}
+        name={name}
+        type="checkbox"
+        checked={checked}
+        className={cls(`default-check-box ${customStyleName}`, className)}
+        disabled={disabled}
+        onChange={e => changeValue(e.currentTarget.checked)}
```

Comment:
i did same mistake with fields before this one. if you gave default `onChange` function in here, every checkbox component turns to controlled. we can use uncontrolled checkbox too sometimes. also, we're using formik and formik can't validate value if there is defined onChange function in component.

can you give `onChange={onChange}` instead of give function please?